### PR TITLE
GUI Animations

### PR DESCRIPTION
**Pull request by noerio**
Add animations when aliens are incanting

### CODE REVIEW

File: zappy_gui/src/NetworkManager/NetworkManager.cpp [hunk 1]
```diff
@@ -266,6 +269,34 @@ void zp::NetworkManager::disconnect(Map &map, Chat &chat)
...
+
+void zp::NetworkManager::getIncantationEnd(const std::vector<std::string> &tokens, zp::Map &map)
+{
+    auto aliens = map.getAliens();
+    sf::Vector2i myPos;
+    sf::Vector2i alienPos(std::stoi(tokens[1]), std::stoi(tokens[2]));
+    zp::Alien *newAlien;
+
+    for (auto alien : aliens) {
+        newAlien = static_cast<zp::Alien *>(alien.get());
```
Comment (Sinan-Karakaya) [inline zappy_gui/src/NetworkManager/NetworkManager.cpp]: Do not use raw pointers

**Review by Sinan-Karakaya — CHANGES_REQUESTED** [review]
Fix raw pointers, replace by unique ptr

**Review by thomasjuin1 — APPROVED** [review]
Good animation 👍, love it

**Review by Sinan-Karakaya — APPROVED** [review]
looks good

**Comment by Sinan-Karakaya** [comment]
DO NOT MERGE

PR might introduce memory leaks, testing in progress

**Review by Sinan-Karakaya — CHANGES_REQUESTED** [review]
Memory leak testing in progress

**Review by Sinan-Karakaya — APPROVED** [review]
Valgrind looks ok

**Review by RoheeAxel — APPROVED** [review]
Seem nice

**Pull request merged**